.github/workflows/docs-issue.yml

```diff
@@ -1,43 +1,41 @@
 # **what?**
-# Open an issue in docs.getdbt.com when a PR is labeled `user docs`
+# Open an issue in docs.getdbt.com when an issue is labeled `user docs` and closed as completed
 
 # **why?**
 # To reduce barriers for keeping docs up to date
 
 # **when?**
-# When a PR is labeled `user docs` and is merged.  Runs on pull_request_target to run off the workflow already merged,
-# not the workflow that existed on the PR branch.  This allows old PRs to get comments.
+# When an issue is labeled `user docs` and is closed as completed.  Can be labeled before or after the issue is closed.
 
 
-name: Open issues in docs.getdbt.com repo when a PR is labeled
-run-name: "Open an issue in docs.getdbt.com for PR #${{ github.event.pull_request.number }}"
+name: Open issues in docs.getdbt.com repo when an issue is labeled
+run-name: "Open an issue in docs.getdbt.com for issue #${{ github.event.issue.number }}"
 
 on:
-  pull_request_target:
+  issues:
     types: [labeled, closed]
 
 defaults:
   run:
     shell: bash
 
 permissions:
-    issues: write # opens new issues
-    pull-requests: write # comments on PRs
-
+    issues: write # comments on issues
 
 jobs:
   open_issues:
-    # we only want to run this when the PR has been merged or the label in the labeled event is `user docs`.  Otherwise it runs the
+    # we only want to run this when the issue is closed as completed and the label `user docs` has been assigned.
+    # If this logic does not exist in this workflow, it runs the
     # risk of duplicaton of issues being created due to merge and label both triggering this workflow to run and neither having
     # generating the comment before the other runs.  This lives here instead of the shared workflow because this is where we
     # decide if it should run or not.
     if: |
-      (github.event.pull_request.merged == true) &&
-      ((github.event.action == 'closed' && contains( github.event.pull_request.labels.*.name, 'user docs')) ||
+      (github.event.issue.state == 'closed' && github.event.issue.state_reason == 'completed') && (
+      (github.event.action == 'closed' && contains(github.event.issue.labels.*.name, 'user docs')) ||
       (github.event.action == 'labeled' && github.event.label.name == 'user docs'))
```

Comment:
do we need to emulate what we're doing in other places? https://github.com/dbt-labs/dbt-redshift/pull/847/files#diff-9b299312ba5915fe644f25c3ea9d363d232b089664545518e8e020720cc21baeR35

Reply:
okay on re-read I think we can just delete line 37 right?

Reply:
@mikealfare  any thoughts on this, I believe this was added to cover your suggestion that we may also want to handle instances of labels being able to kick up docs issue before in case the context is needed to be built up over time instead of after completion.

Reply:
If we're only catching the close event (`github.event.action == 'closed'`) then we should update the `types` to only list `closed` instead of `[closed, labeled]`. If we also want to support adding the label after the issue is closed, then we need to keep the labeled event in `types` and update the logic here for that scenario. The fastest would be to remove `labeled` from `types` and leave that as a follow-up when/if we want to do it.

Reply:
Ok I just misunderstood the logic here, I think we should support adding a label after the issue is closed. This is the kind of papercut we'll just never get back to otherwise